packages/ui/src/utils/TypesUtil.ts

```diff
@@ -10,3 +10,27 @@ export type Color =
   | 'success-100'
 
 export type Size = 'inherit' | 'lg' | 'md' | 'sm' | 'xs' | 'xxs'
+
+export type Spacing = '3xl' | '3xs' | '4xs' | 'l' | 'm' | 's' | 'xl' | 'xs' | 'xxl' | 'xxs'
+
+export type BorderRadius = Exclude<Spacing, 'xl' | 'xxl'>
+
+export type FlexDirection = 'column-reverse' | 'column' | 'row-reverse' | 'row'
```

Comment:
We can improve all of these and avoid discrepancies with PresetsUtil by doing something like

```
export type FlexWrap = typeof flexWrapOptions[number]
```

P.S for this to work you might need to do something like `flexWrapOptions as const`

Here is related stackoverflow post https://stackoverflow.com/a/54061487/911930